annotators/DeepPavlovFactoidClassification/README.md

```diff
@@ -0,0 +1,9 @@
+# Title
+## Description
+
+## Input/Output
+
+**Input**
+**Output**
```

Comment:
?

Reply:
For this one need to ask someone if/how this one is different from the Factoid clf in `combined classification` (the info on this one is scarce in our repo). 
Pinged in our slack channel.